README.md

```diff
@@ -29,17 +29,25 @@ compile "io.arrow-kt:arrow-effects-instances:$arrow_version"
 compile "io.arrow-kt:arrow-instances-data:$arrow_version"
 ```
 
-To enable Arrow in your project include these dependencies in the `dependencies` section in build.gradle file:
-the run:
+To enable Arrow in your project include these dependencies in the `dependencies` section in build.gradle file. Then run the following command:
 
 ```groovy
 ./gradlew clean build
 ```
 
+Don't worry about the **compile time warnings and errors**. It's intended, the exercises are all marked as `TODO()` which means they're not implemented yet and therefore making tests fail. Hopefully you'll be the one fixing those! 
```

Comment:
When going through this as I user I found that after doing the very first build there were build time errors that could confuse the attendees. I think this clarification provides value in that matter.